java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2268,7 +2269,7 @@ public void testZeroVectorEquals() {
   @Test
   public void testZeroVectorNotEquals() {
     try (final IntVector intVector = new IntVector("int", allocator);
-         final ZeroVector zeroVector = new ZeroVector()) {
+        final ZeroVector zeroVector = new ZeroVector()) {
```

Comment:
Not a big deal, but the alignment seemed right to me before so that final is after the above parenthesis. I see the file has both ways, so no need to fix now. I'm not sure why the checkstyle didn't error one way or the other?

Reply:
I think now checkstyle in java code (/dev/checkstyle/checkstyle.xml) is not strong enough to catch some problem like you mentioned above and something like 'more than one empty line'. I think we could to take a close watch to see if it‘s possible to enhance this.